api/database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -17,6 +17,7 @@ public function up(): void
             $table->string('last_name', 128);
             $table->string('email', 512)->unique();
             $table->text('password');
```

Comment:
Must the password be text?

Reply:
Because we don't know exactly hashed password's length, so text is better than string with defined max length.